src/js/server/attendance/email-templates.js

```diff
@@ -23,4 +23,18 @@ module.exports = {
       },
     };
   },
+  expiry({ name, daysValid }) {
+    return {
+      subject: 'Your Hack Cambridge invitation has expired',
+      body: {
+        name,
+        intro: [
+          `Earlier we sent you an invitation to Hack Cambridge Recurse with ${daysValid} days to respond. We have not received a response from you and your invitation has expired.`,
+          'We hope to see you apply for the next Hack Cambridge!'
```

Comment:
We don't need it, just that it says go to our website so would be nice to be able to link to it.